### PR TITLE
Fix Gradient Accumulation Logic

### PR DESCRIPTION
**Pull request by waitzkin**
This fixes the gradient accumulation logic in finetune_full.py.

Since weight update should occur once in every {gradient_accumulation_steps} iterations, I think "==" in line 100 should be "!=".

I would appreciate any comments about this :)

### CODE REVIEW

**Review by lantiga — APPROVED** [review]
Great catch @waitzkin, we got this right in the pretraining script but not here.
